Gemfile

```diff
@@ -46,6 +46,7 @@ gem 'awesome_print'
 
 # Report Generator
 gem 'rtf'
+gem 'sablon', github: 'ComPlat/sablon', branch: "ADD_SUPPORT_TO_IMAGES"
```

Comment:
In the original repository, the branch has not been merged. (I guess the owner is busy)
So I keep this the same status as the original one.

We can commit it to master at Complat repository, depending on the rule for this.